### DISCUSSION TITLE
Welcome to PocketOptionAPI Discussions!

### DISCUSSION THREAD

**Discussion by theshadow76** [ChipaDevTeam/PocketOptionAPI#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Rufus011** [comment]
Напишите инструкцию по установке PocketOptionAPI! на python.

Write instructions on how to install PocketOptionAPI! in python.

**Comment by Elia415** [comment]
Hello!! How can I install the API in Python?

**Reply by theshadow76** [reply]
Talk to us in discord

**Reply by Rufus011** [reply]
Привет!!! у меня есть проект для Pocket option но я не могу установить ключ Api этой платформы, могли бы вы предоставить подробный мануал как это сделать ,буду очень благодарин, поиски по просторам интернета результатов не дали , нет не видео как это делать не инструкции, помогите пожалуйста...

**Reply by theshadow76** [reply]
Talk to us in discord

RU: поговори с нами в дискорде

**Reply by Rufus011** [reply]
не могу отвечать или писать в дискорте не пойму почему

**Reply by Rufus011** [reply]
![Снимок экрана 2024-06-17 190159](https://github.com/theshadow76/PocketOptionAPI/assets/132999068/80d240f2-371a-4c01-bc57-43860411d9b6)
